gtfu/src/main/java/gtfu/tools/GitHubUtil.java

```diff
@@ -85,8 +85,9 @@ private void createCommit(String branchName, String message, String path, byte[]
         GHRef branch = repo.createRef("refs/heads/" + branchName, commit.getSHA1());
     }
 
-    private void createPR(String title, String branchName, String description) throws Exception {
+    private void createPR(String title, String branchName, String description, Boolean autoMerge) throws Exception {
```

Comment:
As mentioned, `Boolean` is a wrapper class for the primitive `boolean` type. Unless there are very specific reasons  please use the primitive type.